scmrepo/git/backend/base.py

```diff
@@ -25,6 +26,12 @@ def __init__(self, func):
         super().__init__(f"No valid Git backend for '{func}'")
 
 
+class SyncStatus(Enum):
+    SUCCESS = 0
+    DUPLICATED = 1
```

Comment:
```suggestion
    UP_TO_DATE = 1
```

I think it's better to use the same terms as git's UI here